app/models/Service.ts

```diff
@@ -0,0 +1,93 @@
+import { get } from '../utils/DataService';
+import { parseAPISchedule } from '../utils/transformSchedule';
+import { Organization } from './Organization'; // eslint-disable-line import/no-cycle
+import {
+  Address,
+  Category,
+  Eligibility,
+  Note,
+  Program,
+  Schedule,
+} from './Meta';
+
+// A Service is provided by an Organization
+export interface Service {
+  id: number
+  name: string
+  addresses?: Address[]
+  alsoNamed: string
+  alternate_name?: string|null
```

Comment:
I only added `?` or `| null` case by case as they caused type errors or were returned null, but just tried removing the ? and it works now, so must have resolved.

Generally, the object is created omitting params, but the API returns everything as null, so it is definitely possible, though can type as Partial<Service> in those cases if needed, although you lose checking for required params at that point, so there is probably a use case for optional and nullable params.

Reply:
> so there is probably a use case for optional and nullable params.

Yeah, I came that that realization later on in my review, where I suggested that maybe we just need separate types for the objects coming back from the API and objects we are sending to the API, since a property that's optional for making an API request might not be optional when coming back, especially if the API provides some non-null default.